shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -833,9 +834,9 @@ public AccessibilityNodeProvider getAccessibilityNodeProvider() {
    * Prior to Android Q, it's impossible to add real views as descendants of virtual nodes. This
    * breaks accessibility when an Android view is embedded in a Flutter app.
    *
-   * <p>This method overrides a @hide method in {@code ViewGroup} to workaround this limitation.
-   * This solution is derivated from Jetpack Compose, and can be found in the Android source code as
-   * well.
+   * <p>This method overrides a This member is not intended for public use, and is only visible for
+   * testing. method in {@code ViewGroup} to workaround this limitation. This solution is derivated
+   * from Jetpack Compose, and can be found in the Android source code as well.
```

Comment:
is this right?

Reply:
Hmm. I'll look at this.

The problem is `@hide` is not wsupported by javadoc. I did a mass find and replace of that with this text, but this looks wrong.

Reply:
Changing this `@hide` to `a hidden` so that the linter won't complain about an invalid keyword.